man/chfn.1.xml

```diff
@@ -80,7 +81,10 @@
       numbers. The <emphasis remap='I'>other</emphasis> field is used to
       store accounting information used by other applications.
     </para>
-
+    <para>
+      If <replaceable>LOGIN</replaceable> is not specified, <command>chfn</command>
+      prompts for the current user account.
```

Comment:
I can't use chfn from shadow as my distribution uses util-linux, but I have a doubt. Does it prompt for the user account? Or does it assume the current user account?

Reply:
It seems to me that it assumes the current user:

```sh
alx@devuan:~$ which chfn | xargs dpkg -S | cut -f1 -d:
passwd
```
```sh
alx@devuan:~$ chfn
Password: 
chfn: PAM: Authentication failure
```

Reply:
@zeha do you mind updating this to reflect that?

Reply:
I can do it.  @zeha is just forwarding the diffs that other people proposed to Debian, and I promised to transform them into proper patches.